optimum/intel/neural_compressor/quantization.py

```diff
@@ -398,6 +398,12 @@ def _weight_only_quantization(
     if (not torch.cuda.is_available() or device_map == "cpu") and model.config.model_type == "chatglm":
         model = model.float()
 
+    from neural_compressor.torch import load_empty_model
+
+    model = load_empty_model(
+        model_id,
+        trust_remote_code=trust_remote_code,
+    )
```

Comment:
The model was already loaded above so would make sense to remove this part as well, alos we need to set `cls=load_empty_model` when loading the model with load_empty_model https://github.com/intel/neural-compressor/blob/v3.1.1/neural_compressor/torch/utils/utility.py#L354 
```suggestion
    model = load_empty_model(model_id, cls=model_class, **loading_kwargs)
```

Reply:
I improve the code, load_empty_model only needed by "layer-wise" feature.

Reply:
I agree, thanks suggestion. I have improved it in INC(Here is the PR https://github.com/intel/neural-compressor/pull/2079, I also update the commit id in optimum-intel setup.py), `load_empty_model` has support pass `**loading_kwargs` included `token` or `subfolder` now.